enhancements/monitoring/hypershift-monitoring.md

```diff
@@ -0,0 +1,392 @@
+
+For additional acronyms, see [Red Hat Dictionary](https://source.redhat.com/groups/public/red-hat-dictionary-or-lexicon/red_hat_abbreviations_acronyms_and_initialisms_dictionary_wiki)
+
+## Background
+
+[HyperShift project](https://hypershift-docs.netlify.app/reference/controller-architecture/) brings some changes to how OpenShift clusters can be deployed. The 10,000 feet view is to decouple the OCP control plane from the data plane. Control plane components (e.g. etcd, API servers, …) run on an underlay cluster, while worker components run remote on a different infrastructure.
+
+This is similar to how AWS, Google and so on run their managed Kubernetes service: You provide the worker nodes and they provide the control plane.
+
+The core idea of HyperShift topology is that it allows sharing infrastructure on control planes and allows the “hosted clusters” (workers) to have a lower computation and operational overhead.
+
+### Important HyperShift Assumptions
+
+* One-way communication. HyperShift management cluster can “talk to” the data plane, but not the opposite (except Kube API, Ignition and OIDC endpoint).
+* HyperShift is managed first, but form factor will be offered for self-management by 4.11.
+* There is tight budget on computation for control planes (outside of MVP).
+
+## Motivation
+
+This document enhancement proposes potential solution for Admin and Customer metric monitoring within those new topologies, since we cannot apply easily our current platform monitoring products using within cluster monitoring operator (CMO) easily (see [Alternatives](#use-pure-platform-monitoring-solution) on why).
+
+### Goals
+
+The goal is to provide a solution that maintains similar monitoring capabilities we offer with single box monitoring using CMO in HyperShift for our three [personas](#personas).
+
+> NOTE: The definition of “monitor” in the bullet list below means ability to graph, query and alert on near-real-time metrics about involved components.
+
+* HSRE can monitor management clusters and control planes running there.
+  * Ability to use AlertManager and PagerDuty workflow
+  * Ability to monitor Tugboats themselves
+  * Ability to monitor Customer control planes
+  * Ability to view fleet-level metrics and drill-downs in a single pane of glass
+  * Ability to keep metrics in-region
+* HSRE can monitor control-plane related (managed) software on guest clusters:
+  * For example Ingress/Router/CNI/CSI 
+  
+> Exact list of what is scraped on guest clusters for HSRE does not matter. All we need to know: some service monitors.
+
+* HSRE can query metrics data over SLO period (usually 28 days).
+  * Longer durations nice to have, but technically not required. There is also option that allows HSRE to persist SLO from platform to [different backend if needed](https://docs.google.com/document/d/1BJarERppgiJ8esc6d8anbJMOQ0AflFBQea-Zc9wAp0s/edit?disco=AAAAQTFsbwM).
```

Comment:
We can't miss a step here.  28 days is bare minimum but really we do want longer.  I'd argue one year.  Realize that discussion is out of scope for this proposal.

Reply:
It is, but I think we are fine with one year out of box for you